zeppelin-web/test/karma.conf.js

```diff
@@ -18,15 +18,42 @@ module.exports = function(config) {
 
     // list of files / patterns to load in the browser
     files: [
```

Comment:
Might want to checkout wiredep for all those bower_components.
It is already taking care of adding .js dependencies in the index.html, so I'm pretty sure it could add it here too